src/cascadia/TerminalApp/TerminalPaneContent.cpp

```diff
@@ -235,6 +235,18 @@ namespace winrt::TerminalApp::implementation
             co_return;
         }
 
+        // We try to read the setting from the command line arguments
+        if (const auto& settings{ _cache.TryLookup(_profile) })
+        {
+            const auto closeMode = settings.DefaultSettings().CloseOnExit();

```

Comment:
@zadjii-msft 
I have updated the code to handle the command line setting now. But it doesn't work because `closeMode` is always `Automatic` and never the updated value after command line parsing. How can I get the updated value that I manipulate in the command line parser?

Reply:
Opened issue #17473.